aion/web/controllers/question_controller.ex

```diff
@@ -17,12 +19,16 @@ defmodule Aion.QuestionController do
       question_changeset = Question.changeset(%Question{}, Map.put(question_params, "belongs_to", question_subject))
       case Repo.insert(question_changeset) do
         {:ok, question} ->
-          answer_changeset = Answer.changeset(%Answer{}, %{"content" => answers_content, "belongs_to" => question})
-          case Repo.insert(answer_changeset) do
-            {:ok, answer} ->
-              question
-            {:error, error_answer_changeset} ->
-              Repo.rollback(error_answer_changeset)
+          insert_asnwers =
+            answers_content
+            |> String.split(@answer_separator)
+            |> Enum.map(fn answer_content -> Answer.changeset(%Answer{}, %{"content" => answer_content, "belongs_to" => question}) end)
```

Comment:
I somewhat agree since these steps are significant, however it scatters the implementation and this is kind of debatable if your suggestion is more legible.